internal/file/file_manager_service_test.go

```diff
@@ -58,3 +62,191 @@ func TestFileManagerService_UpdateFile(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, 1, fakeFileServiceClient.UpdateFileCallCount())
 }
+
+func TestFileManagerService_ConfigApply_Add(t *testing.T) {
+	ctx := context.Background()
+	tempDir := t.TempDir()
+	addAction := mpi.File_FILE_ACTION_ADD
+
+	filePath := filepath.Join(tempDir, "nginx.conf")
+	fileContent := []byte("location /test {\n    return 200 \"Test location\\n\";\n}")
+	fileHash := files.GenerateHash(fileContent)
+	defer helpers.RemoveFileWithErrorCheck(t, filePath)
+
+	overview := mpi.FileOverview{
```

Comment:
can you add FileOverview to the test helpers? you might need to pass filePath and fileHash